src/components/ae-address/ae-address.vue

```diff
@@ -72,8 +72,23 @@ export default {
      * accepts any valid css value, ex: `10px, 1rem, 50%, 10px 10px`
      */
     gap: String,
+
+    /**
+     * When set to `true` enables copy to clipboard
+     * when the user clicks/taps the address
+     */
+    copyOnClick: Boolean,
```

Comment:
I agree on the name, I'll update it. As for the default behaviour, I believe disabled would be the main behaviour, since you don't want to enable that feature everywhere, and also it needs some style update.